prediction-polls/backend/src/services/AuthGoogleService.js

```diff
@@ -0,0 +1,93 @@
+const qs = require("querystring");
+const axios = require("axios");
+const db = require("../repositories/AuthorizationDB.js");
+
+const { generateAccessToken, generateRefreshToken } = require('./AuthorizationService.js');
+
+
+async function googleLogIn(req,res){
+
+    console.log("GOOGLE REQUEST IS RECIEVED")
```

Comment:
Development tools shouldnt be added to production. Since the development is complete you should omit logs. If you have good reason please tell me.